doc/site/docs/02-core-concepts/06-body.md

```diff
@@ -86,9 +86,43 @@ final streamBody = Body.fromDataStream(
 );
 ```
 
+### Static file serving
```

Comment:
This shouldn't really be part of the `body.md`. I think we need its own page for static files. It's not clear this is the place to find it by looking at the file structure. This can potentially be another PR, but it should be fixed.

Reply:
Will do this in a new PR